apps/epic-web/src/templates/workshop-template.tsx

```diff
@@ -309,8 +310,10 @@ const Header: React.FC<{module: Module; canView: boolean}> = ({
           <div className="w-full pt-8 text-lg">
             <div className="flex items-center justify-center gap-3 md:justify-start">
               <ResourceContributor
-                name={'Kent C. Dodds'}
-                slug={'kent-c-dodds'}
+                name={module.instructor.name || 'Kent C. Dodds'}
+                slug={module.instructor?.slug || 'kent-c-dodds'}
+                image={module.instructor?.picture?.url}
+                as="div"
```

Comment:
@vojtaholik suggested I pass `as="div"` so that the whole component would be clickable